### ISSUE TITLE
HTML-validation error with `aria-owns`

### ISSUE THREAD

**Issue by boris-petrov** [quill-mention/quill-mention#418]
Thank you for the great project!

When I started using it, my HTML-validator complained that `The “aria-owns” attribute must point to an element in the same document.`. Which is true - unless `quill-mention` has shown something, there is an `aria-owns` attribute with the value `quill-mention-list`. Which is invalid HTML.

Another issue the validator complains about is when the list is shown and there is an `aria-activedescendant` attribute with value `quill-mention-item-0`. Then the complaint is `Element “div” is missing one or more of the following attributes: “aria-expanded”, “role”.` I'm not sure why these are needed but I guess it's correct.

**Comment by csculley** [comment]
Can I ask which validator you're using? It seems really nice!

**Comment by boris-petrov** [comment]
[Nu Html Checker](https://github.com/validator/validator/). :)
